allensdk/brain_observatory/behavior/behavior_project_cache/behavior_project_cache.py

```diff
@@ -57,7 +57,10 @@ class VBOLimsCache(Cache):
     }
 
 
-class VisualBehaviorOphysProjectCache(object):
+class VisualBehaviorOphysProjectCache(ProjectCacheBase):
+
+    PROJECT_NAME = "visual-behavior-ophys-data"
+    BUCKET_NAME = "visual-behavior-ophys"
```

Comment:
You've swapped the bucket name and the project name. As a test, this script should run (it just loads a cache that I instantiated from VBO before we started VBN work)

```
import os
cache_dir = '/allen/aibs/informatics/danielsf/scratch/example_cache'

from allensdk.brain_observatory.behavior.behavior_project_cache.behavior_project
_cache import VisualBehaviorOphysProjectCache as cache_class

cache = cache_class.from_s3_cache(cache_dir=cache_dir)
```

Currently, it fails with a "bucket does not exist" error. You will probably want to swap the bucket and project names for VBN as well (though, we obviously haven't settled on a name for anything yet).

Reply:
good catch